Appraisals

```diff
@@ -66,6 +66,7 @@ elsif '1.9.3' <= RUBY_VERSION && RUBY_VERSION < '2.0.0'
       gem 'sinatra', '1.4.5'
       gem 'sqlite3'
       gem 'activerecord', '3.2.22.5'
+      gem 'database_cleaner'
```

Comment:
If we're going to add `database_cleaner` to the test suite, because it helps us run tests, then this should become a `development_dependency` in the gemspec, instead of being added here to Appraisals.  This file and section is for external libraries we test for compatibility with `ddtrace`, not test dependencies.